### PR TITLE
Reducing app size and warning by removing unused code

### PR DESCRIPTION
**Pull request by RiishavRG**
Removed code snippets that contribute to app size and warnings:
1. Removing version check -  which would result in not being notified when swift-protobuf bumps the ProtobufAPIVersion which is okay as it would give compile time errors
2. Removing == function from StorageClass as it is not required internally and by us.
3. Removing generation of has and clear functions in protos as it is not used internally and by us.

### CODE REVIEW

File: Sources/protoc-gen-swift/MessageFieldGenerator.swift [hunk 1]
```diff
@@ -116,7 +116,7 @@ class MessageFieldGenerator: FieldGeneratorBase, FieldGenerator {
                 p.print("\(comments)\(visibility)var \(swiftName): \(swiftStorageType) = \(swiftDefaultValue)")
             }
         }
-
+        return
```
Comment (asifmohd) [inline Sources/protoc-gen-swift/MessageFieldGenerator.swift]: Can we delete the rest of the code here and remove the return?

File: Sources/protoc-gen-swift/FileGenerator.swift [hunk 2]
```diff
@@ -122,7 +122,7 @@ class FileGenerator {
         }
 
         p.print()
-        generateVersionCheck(printer: &p)
+        // generateVersionCheck(printer: &p)
```
Comment (asifmohd) [inline Sources/protoc-gen-swift/FileGenerator.swift]: Let's delete instead of commenting it out.

Later on we can use git diff to compare changes.

Comment (RiishavRG) [reply Sources/protoc-gen-swift/FileGenerator.swift]: yes, updated it

File: Sources/protoc-gen-swift/MessageFieldGenerator.swift (hunk 1 above)
Comment (RiishavRG) [reply Sources/protoc-gen-swift/MessageFieldGenerator.swift]: yes, updated it

**Pull request merged**